Code/GraphMol/MolAlign/Wrap/rdMolAlign.cpp

```diff
@@ -661,6 +695,59 @@ BOOST_PYTHON_MODULE(rdMolAlign) {
       docString.c_str());
 
   docString =
+      "Returns the RMS between two conformations.\n\
+     \n\
+     By default, the conformers will be aligned to the first conformer\n\
+     of the molecule (i.e. the reference) before RMS calculation and,\n\
+     as a side-effect, will be left in the aligned state.\n\
```

Comment:
I think you mean here: "as a side-effect the conformers will be left in aligned state"